snippets/fsharp/VS_Snippets_CLR_System/system.console.class/fs/example3.fs

```diff
@@ -0,0 +1,169 @@
+﻿
+// dump a range of Unicode characters as a 16x16 array
+// <Snippet4>
+open System
+open System.IO
+open System.Globalization
+open System.Text
+
+type uint = uint32
```

Comment:
This is fine, but typically type abbreviations aren't used like this. I've made the following language suggestion: https://github.com/fsharp/fslang-suggestions/issues/818

Reply:
I kept this in place for now but I can change it if you think it is bad form for a document sample to use a type abbreviation.